hmda/build.sbt

```diff
@@ -20,7 +30,21 @@ dockerCommands :=
 
 javaOptions in Universal ++= Seq(
   "-J-XX:+UnlockExperimentalVMOptions",
-  "-J-XX:+UseCGroupMemoryLimitForHeap"
+  "-J-XX:+UseContainerSupport",
+  "-J-XX:+UnlockDiagnosticVMOptions",
+  "-J-XX:+DebugNonSafepoints",
+  "-J-Xloggc:/opt/docker/gc.log",
+  "-J-Xlog:gc*",
+  "-J-XX:+HeapDumpOnOutOfMemoryError",
+  "-J-XX:HeapDumpPath=/dumps/oom.bin",
+  "-Dcom.sun.management.jmxremote",
```

Comment:
Do we need these `jmxremote` settings in production?

Reply:
It is good to have the option of being able to profile the application if we need to instead of re-deploying just to enable profiling.

Reply:
Do they work with jdk-slim?

Reply:
Yup, you can port-forward on 1099 for a JMX connection via visualvm